tests/docker/BUILD

```diff
@@ -285,6 +304,12 @@ file_test(
     file = ":push_test.digest",
 )
 
+file_test(
+    name = "test_push_tag_file_digest_output",
```

Comment:
Don't check the digest output, check the default output executable that it contains the right tag.